<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="panelLayout">
      <item>
       <layout class="QHBoxLayout" name="potLayout">
        <item>
         <widget class="QLabel" name="potLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Potential:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="potComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Harmonic x^2</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="xLayout">
        <item>
         <widget class="QLabel" name="xLabel">
          <property name="text">
           <string>x:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="xSlider">
          <property name="minimum">
           <number>-200</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="pageStep">
           <number>20</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="vLayout">
        <item>
         <widget class="QLabel" name="vLabel">
          <property name="text">
           <string>v:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="vSlider">
          <property name="minimum">
           <number>-200</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="pageStep">
           <number>20</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="tLayout">
        <item>
         <widget class="QLabel" name="tLabel">
          <property name="text">
           <string>dt:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="tSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>-200</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="pageStep">
           <number>20</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="outputTextBrowser">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>447</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>918</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>potComboBox</tabstop>
  <tabstop>xSlider</tabstop>
  <tabstop>vSlider</tabstop>
  <tabstop>tSlider</tabstop>
  <tabstop>outputTextBrowser</tabstop>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>17</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateDisplay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateDisplay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateDisplay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>potComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateDisplay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateDisplay()</slot>
 </slots>
</ui>
